test/storage.test.js

```diff
@@ -1,99 +1,107 @@
     expect(result).toContain(feedId2.toString());
   });
+});
 
-  const post = {
-    guid: 'tag:blogger.com,1999:blog-7100164112302197371.post-522285656016053350',
-    author: 'Neil David',
-    title: 'My First Blog',
-    link: 'http://nadavid.blogspot.com/2008/09/my-first-blog.html',
-    content:
-      "I have never done this before, so let's make this short. This post is just a test on how my blog post will look like.",
-    text: 'post one text',
+describe('Tests for posts', () => {
+  const testPost = {
+    guid: '0',
+    author: 'foo',
+    title: 'foo',
+    link: 'foo',
+    content: 'foo',
+    text: 'foo',
     updated: new Date('2009-09-07T22:23:00.544Z'),
     published: new Date('2009-09-07T22:20:00.000Z'),
-    url: 'http://seneca.co/jsmith',
-    site: 'wordpress.com',
+    url: 'foo',
+    site: 'foo',
   };
 
-  const post2 = {
-    guid: '2tag:blogger.com,1999:blog-7100164112302197371.post-522285656016053350',
-    author: 'Neil David2',
-    title: 'My First Blog2',
-    link: 'http://nadavid2.blogspot.com/2008/09/my-first-blog.html',
-    content:
-      "I have never done this before, so let's make this short. This post is just a test on how my blog post will look like.",
-    text: 'post one text2',
-    updated: new Date('2008-09-07T22:12:00.544Z'),
-    published: new Date('2008-09-07T22:09:00.000Z'),
-    url: 'http://seneca.co/jsmith',
-    site: 'wordpress.com',
+  const testPost2 = {
+    guid: '1',
+    author: 'foo',
+    title: 'foo',
+    link: 'foo',
+    content: 'foo',
+    text: 'foo',
+    updated: new Date('2009-09-07T22:23:00.544Z'),
+    published: new Date('2009-09-07T22:20:00.000Z'),
+    url: 'foo',
+    site: 'foo',
   };
 
-  const post3 = {
-    guid: '3tag:blogger.com,1999:blog-7100164112302197371.post-522285656016053350',
-    author: 'Neil David3',
-    title: 'My First Blog3',
-    link: 'http://nadavid2.blogspot.com/2008/09/my-first-blog.html',
-    content:
-      "I have never done this before, so let's make this short. This post is just a test on how my blog post will look like.",
-    text: 'post one text3',
-    updated: new Date('2008-09-07T22:12:00.544Z'),
-    published: new Date('2008-09-07T22:09:00.000Z'),
-    url: 'http://seneca.co/jsmith',
-    site: 'wordpress.com',
+  const testPost3 = {
+    guid: '2',
+    author: 'foo',
+    title: 'foo',
+    link: 'foo',
+    content: 'foo',
+    text: 'foo',
+    updated: new Date('2009-09-07T22:23:00.544Z'),
+    published: new Date('2009-09-07T22:20:00.000Z'),
+    url: 'foo',
+    site: 'foo',
   };
 
+  beforeAll(() => {
+    Promise.all([testPost, testPost2, testPost3].map(post => addPost(post)));
+  });
+
   it('should allow retrieving a post by guid after inserting', async () => {
-    await storage.addPost(post);
-    const result = await storage.getPost(post.guid);
-    expect(result.link).toEqual(post.link);
+    const result = await getPost(testPost.guid);
+    expect(result.link).toEqual(testPost.link);
```

Comment:
All `link` properties above are `'foo'`, so this probably isn't a useful test.  Let's compare `result.guid`, since they are always different.